### PR TITLE
Examples: Integrate lib preview 0.2.0

### PR DESCRIPTION
**Pull request by tbouffard**
Relates to https://github.com/process-analytics/bpmn-visualization-js/pull/479
Integrate the new ES Module bundle and adapt the examples accordingly.
**Important**: the example html page cannot be opened locally without using a web server, the doc has been updated about that

To test live:
- [demo](https://cdn.statically.io/gh/process-analytics/bpmn-visualization-examples/integrate_lib_preview/demo/index.html)
- [examples](https://cdn.statically.io/gh/process-analytics/bpmn-visualization-examples/integrate_lib_preview/examples/index.html)

Remaining tasks
- [x] update `update-lib-version.bash`: consider imports
- [x] explain how to run locally (need http server because of the module usage, otherwise CORS errors)
- [x] integrate demo code build from the lib master branch

### CODE REVIEW

File: scripts/update-lib-version.bash [hunk 1]
```diff
@@ -19,10 +19,12 @@ echo "Search for files in $(pwd)"
 # lines to update start with something like "<script src="../../demo/0.1.6/"
 if [[ "$OSTYPE" == "darwin"* ]]; then
 	sed -i '' -E "s#<script src=\"../../demo/[^/]+/#<script src=\"../../demo/$NEW_VERSION/#" **/*.{html,md}
+	sed -i '' -E "s/\/demo\/[0-9].[0-9].[0-9]/\/demo\/$NEW_VERSION/#" **/*.js
 else
 	sed -i -E "s#<script src=\"../../demo/[^/]+/#<script src=\"../../demo/$NEW_VERSION/#" **/*.{html,md}
+	sed -i -E "s/\/demo\/[0-9].[0-9].[0-9]/\/demo\/$NEW_VERSION/#" **/*.js
```
Comment (tbouffard) [inline scripts/update-lib-version.bash:25]: @aibcmars I have tested this on Ubuntu it works, but
- the `utils.js` is not changed
- we can remove the sed using the `<script src` pattern and keep only the new sed command you added: if we add the html and md pattern, this single sed command update all files

File: scripts/update-lib-version.bash [hunk 2]
```diff
@@ -16,13 +16,11 @@ EXAMPLES_DIRECTORY="$SCRIPT_DIRECTORY/../examples"
...
-# lines to update start with something like "<script src="../../demo/0.1.6/"
+# lines to update contains substring like "/demo/0.1.6/"
 if [[ "$OSTYPE" == "darwin"* ]]; then
-	sed -i '' -E "s#<script src=\"../../demo/[^/]+/#<script src=\"../../demo/$NEW_VERSION/#" **/*.{html,md}
-	sed -i '' -E "s/\/demo\/[0-9].[0-9].[0-9]/\/demo\/$NEW_VERSION/#" **/*.js
+	sed -i '' -E "s/\/demo\/[0-9].[0-9].[0-9]/\/demo\/$NEW_VERSION/#" **/*.{html,md,js} *.{html,js}
 else
-	sed -i -E "s#<script src=\"../../demo/[^/]+/#<script src=\"../../demo/$NEW_VERSION/#" **/*.{html,md}
-	sed -i -E "s/\/demo\/[0-9].[0-9].[0-9]/\/demo\/$NEW_VERSION/#" **/*.js
+	sed -i -E "s/\/demo\/[0-9].[0-9].[0-9]/\/demo\/$NEW_VERSION/#" **/*.{html,md,js} *.{html,js}
```
Comment (tbouffard) [inline scripts/update-lib-version.bash:23]: :+1: nice, thanks

File: scripts/update-lib-version.bash (hunk 1 above)
Comment (tbouffard) [reply scripts/update-lib-version.bash]: fixed with 1fffd4a

**Review by aibcmars — APPROVED** [review]
LGTM

**Pull request merged**